Change jormungandr rest port to 8081

Anviking opened the pull request:
Attempt to fix https://travis-ci.org/input-output-hk/cardano-wallet/jobs/535892104#L303
where we reason that the port 8080 is still busy from the http-bridge integration tests.

piotr-iohk commented on test/manual/scripts/generate-addresses.sh:
```diff
@@ -0,0 +1,30 @@
+  		-d "{
+		  \"accountIndex\": ${ARGS[--account]},
+		  \"walletId\": \"${ARGS[--wallet]}\"
+      }" | jq .data.id)
```
`jq .data.id` - that's cool! :+1:

KtorZ replied:
Not sure if `jq` is installed by default on most Mac OS and Linux distributions but that's definitely a must-have when working with JSON :sweat_smile:

Anviking committed:
Change jormungandr rest port to 8081

Attempt to fix https://travis-ci.org/input-output-hk/cardano-wallet/jobs/535892104#L303
where we reason that the port 8080 is still busy from the http-bridge integration tests.

KtorZ reviewed (approved):
Let's see :man_shrugging:
